packages/react-components/src/molecules/AssociationList.tsx

```diff
@@ -143,11 +144,15 @@ const AssociationList: React.FC<AssociationListProps> = ({
 
                 {inline && <span css={bulletStyles}>·</span>}
               </div>
-              <div css={dividerStyles}>{inline || <Divider />}</div>
+              {inline || (
+                <div css={dividerStyles}>
+                  <Divider />
+                </div>
+              )}
             </li>
           ))}
           {more && (
-            <li>
+            <li data-testid="associationlist-more">
```

Comment:
We don't use data-test-ids in this project. Let me know if you need help figuring out what selector to use

Reply:
Just used here because the assertion wasn't working using the role, the list is divided in two groups and I need to assert first one and then the other to make sure the list is divided, using getByRole asserts all of them at once. How do you think it's a better solution for this?